A66-otel-stats.md

```diff
@@ -144,12 +144,53 @@ the scale to better fit the data.
 This section describes a `CallTracer` approach to collect the client and server
 per-attempt/call metrics. A CallTracer is a class that is instantiated for every
 call. This class has various methods that are invoked during the lifetime of the
-call. On the client side, the CallTracer knows about multiple attempts on the
+call. On the client-side, the CallTracer knows about multiple attempts on the
```

Comment:
Done

Reply:
You added a link in the "related proposal" section, but I think we need to include the word "retry" in the description here somehow.  Otherwise, I'm not sure it's clear to the reader that "attempt" refers to the retry code.

Reply:
added